client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/checklist-item.tsx

```diff
@@ -17,7 +17,8 @@ const ChecklistItem = ( { task }: { task: Task } ) => {
 				data-task={ id }
 				{ ...action }
 			>
-				{ isCompleted && taskDisabled && (
+				{ isCompleted && ( taskDisabled || keepActive ) && (
```

Comment:
At this point, could we remove the `&& ( taskDisabled || keepActive )` condition entirely? 🤔